src/components/IndexTable/IndexTable.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react'
+import { Box, CheckBox, DataTable } from 'grommet'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+const RoundedBox = styled(Box)`
+  border-radius: 0.5em;
+`
+
+class IndexTable extends Component {
```

Comment:
Is this going to be shared between the projects list, workflows list, etc? Perhaps name this `ResourcesTable` or something a bit more descriptive? If it's not shared, then specifically what it is `ProjectsTable`?